libraries/api-documenter/src/index.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as os from 'os';
+import * as path from 'path';
+
+const myPackageJsonFilename: string = path.resolve(path.join(
+  module.filename, '..', '..', 'package.json')
+);
+const myPackageJson: { version: string } = require(myPackageJsonFilename);
```

Comment:
I guess package.json is slightly iffy because the package manager tampers with it.

---
In reply to: [134042183](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134042183) [](ancestors = 134042183,134040687)